src/main/java/auto_batcher/FirstLastName.java

```diff
@@ -0,0 +1,28 @@
+package org.txstate.auto_batcher;
+
+import java.util.*;
+import java.io.*;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FirstLastName
```

Comment:
this is a poor choice of name.
better would have something like `StudentBasicInfo`